data/json/vitamin.json

```diff
@@ -3,39 +3,64 @@
     "id": "calcium",
     "type": "vitamin",
     "name": "Calcium",
-    "min": -600,
-    "deficiency": [ [ "brittlebone", -300 ] ]
+    "deficiency": "hypocalcemia",
+    "min": -720,
+    "disease": [
+      [ -288, -336 ],
+      [ -337, -384 ],
+      [ -385, -720 ]
```

Comment:
It's not more convenient but it's more flexible whilst the design is still being worked on